modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -56,6 +56,10 @@ The API carousel component shows a horizontally scrollable set of API cards.
 
 The interactive API console component shows information about your APIs and their methods, and allows users to test the methods by sending requests.
 
+Endpoints are displayed in alphabetical order, even when the original API specification listed them in a different order.
+
+The console components are the *API Console*, *API Console Documentation*, *API Console Instances*, *API Console Navigation*, and *API Console Request Panel*. In the Builder you can add multiple console components to the same page. If you do this, any change in one console component will also be visible in the other console components on the same page that show the same API. For example, selecting the *Summary* tab in one API Console will select the *Summary* tab in other API Consoles on the same page that show the same API.
```

Comment:
Minor suggestions:
Console components include *API Console*, *API Console Documentation*, *API Console Instances*, *API Console Navigation*, and *API Console Request Panel*. In the Builder, you can add multiple console components to the same page. Changes to a console component are visible to all other console components on that page that display the same API. For example, selecting the *Summary* tab in an API Console results in the selection of the *Summary* tab in all API Consoles on the same page that display the same API.

Reply:
Is there a reason that 'show' is used vs 'display'?

Reply:
Yes, 'display' is good. Made all of these changes.